src/main/resources/config.yml

```diff
@@ -18,16 +18,22 @@ sendMessage: true
 saveState: true
 #
 # Should a Player be on join default immune?
-defaultImmune: false
+defaultImmune: true
```

Comment:
Why should the players default immune at the default config?

Reply:
```suggestion
defaultImmune: false
```